website/pages/component/popover/example.tsx

```diff
@@ -73,11 +72,7 @@ function DefaultExample({
   const [active, setActive] = useState(false);
   return (
     <div>
-      <Button
-        onClick={() => setActive(!active)}
-        variant="primary"
-        className={containerStyle}
-      >
+      <button onClick={() => setActive(!active)} className={containerStyle}>
```

Comment:
won't need this change anymore if https://github.com/mongodb/leafygreen-ui/pull/732 gets merged

Reply:
True!

Reply:
Reverted!